gelf.go

```diff
@@ -89,6 +89,10 @@ func (a *GelfAdapter) Stream(logstream chan *router.Message) {
 			RawExtra: extra,
 		}
 
+		if len(msg.Short) == 0 {
```

Comment:
I wonder if it is better to check `msg.Short == ""`?

Reply:
Fixed. 🙂